py-torch-sparse: add v0.6.17

qwertos opened the pull request.

qwertos committed:
[py-torch-sparse] New version 0.6.17

qwertos committed:
[py-torch-sparse] added dependency on parallel-hashmap

adamjstewart commented on var/spack/repos/builtin/packages/py-torch-sparse/package.py:
```diff
@@ -13,18 +13,21 @@ class PyTorchSparse(PythonPackage):
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.7:", when="@0.6.17:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-pytest-runner", type="build")
```
```suggestion
    depends_on("py-pytest-runner", when="@:0.6.8", type="build")
```

adamjstewart commented on var/spack/repos/builtin/packages/py-torch-sparse/package.py:
```diff
@@ -13,18 +13,21 @@ class PyTorchSparse(PythonPackage):
 
     variant("cuda", default=False, description="Enable CUDA support")
 
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.7:", when="@0.6.17:", type=("build", "run"))
```
Spack only supports Python 3.7+, you could remove both of these lines

qwertos committed:
[py-torch-sparse]

- spack only supports python@3.7:
- py-pytest-runner only needed with old versions
